apb_overview.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+This document outlines the steps for each APB task to help people to understand what is happening under the hood.
+
+# Provision
+
+1. Create a new Postgres service, including:
+    * Secrets for database admin users
+    * Service for the Postgres DB
+    * PersistVolume Claim to for the Postgres DB
+    * DeploymentConfig for the Postgres DB
+2. Create a new Keycloak service, including:
+    * DeploymentConfig for Keycloak
+        * Link Keycloak and Postgres DB
+        * Add default Keycloak admin user credential
+    * Service for Keycloak
+    * Routes (both HTTP and HTTPS) for Keycloak
```

Comment:
This should be either HTTP or HTTPS depending on the `keycloak_protocol` variable.

It defaults to a HTTPS route which will redirect any HTTP requests.